.dockerignore

```diff
@@ -1,9 +1,16 @@
-# Ignore everything except for the specific files we actually care about. This is to avoid sending a
-# massive build context to the docker daemon, and also allows us to avoid copying everything we
-# might need into a temp directory.
-*
```

Comment:
I did a little digging here. With this `.dockerignore` file as it stands now, I've got a ~10MB build context sent to Docker.

That said, one option we could try avoid such micromanagement of this `.dockerignore` would be to use [BuildKit](https://docs.docker.com/develop/develop-images/build_enhancements/). It apparently scans the Dockerfile for which files are _actually_ referenced by that Dockerfile and only sends those to the context. From my tests, this results in the same ~10MB build context when given an empty `.dockerignore` file.

If you'd like to run your own test, you can set `DOCKER_BUILDKIT=1 docker build ...`.

Take this with a grain of salt. I haven't used BuildKit before and I'm not sure why it isn't the default builder yet.

Reply:
> we could try avoid such micromanagement of this .dockerignore

We could also just .dockerignore `parser/target` and not worry about the rest